sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/io/AvroIOTest.java

```diff
@@ -222,6 +222,37 @@ public void testAvroSinkWrite() throws Exception {
     }
   }
 
-  // TODO: for Write only, test withSuffix, withNumShards,
+  @SuppressWarnings("deprecation") // using AvroCoder#createDatumReader for tests.
+  @Test
+  public void testAvroSinkShardedWrite() throws Exception {
```

Comment:
This particular test (and probably others in this file) could be made runner-agnostic. That would make them much better.

However, this is an underlying problem here. Ok to ignore for now.